ui/components/multichain/select-action-modal-item/select-action-modal-item.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  FlexDirection,
+  IconColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarIcon,
+  AvatarIconSize,
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../component-library';
+import Tooltip from '../../ui/tooltip';
+
+export const SelectActionModalItem = ({
+  actionIcon,
+  onClick,
+  showIcon,
+  primaryText,
+  secondaryText,
+  isDisabled,
+  tooltipTitle,
+}) => {
+  const SelectActionItem = () => {
+    return (
+      <Box
+        paddingTop={4}
+        paddingBottom={4}
+        gap={4}
+        display={Display.Flex}
+        flexDirection={FlexDirection.Row}
+        as="a"
```

Comment:
Can this be a `<button>` instead?

Reply:
Or `<ButtonIcon>`?

Reply:
Hmm! So, this component would be replaced by new DS Cell component when it will be ready and it can behave both like `button` or `a` confirmed by @SaraCheikh . For this one, I think it can be both `a` or `button`, because some buttons like `buy` redirects the user to portfolio website. Also, if we convert it to button now, we will have to override the css. So, I think we can keep it `as="a" ` for now. wdyt?